mods/map/init.lua

```diff
@@ -65,7 +71,7 @@ minetest.register_craftitem("map:mapping_kit", {
 minetest.register_craft({
 	output = "map:mapping_kit",
 	recipe = {
-		{"default:glass", "default:paper", "group:stick"},
+		{"default:glass", "default:paper", "default:stick"},
```

Comment:
Ditto

Reply:
Oh this is bad. I was aware of some unrelated changes or deviations earlier and tried to fix this in advance. Obviously, I got it the wrong way and one has to presume some more pollutions.

Please accept my apologies and I presume to wait until you reach 'xpane' apparently.

Reply:
No problem. I'm glad you put effort into this PR, even though Git is not easy to understand at the beginning.